app/views/content_items/_body_with_related_links.html.erb

```diff
@@ -24,3 +24,8 @@
 
   <%= render 'shared/sidebar_navigation' %>
 </div>
+
+<% if @content_item.has_policy_and_engagement_links? %>
```

Comment:
This logic looks good, but it needs to be added to `shared/_taxonomy_navigation.html.erb`